app/services/logger.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const winston = require('winston');
+
+const tsFormat = () => (new Date()).toLocaleTimeString();
+
+const fs = require('fs');
+
+// Create the log directory if it does not exist
+if (!fs.existsSync('./logs')) {
+	fs.mkdirSync('./logs');
+}
+
+const logger = new winston.Logger({
+	transports: [
```

Comment:
Please pull this configuration stuff out into the config files. In our config folder, `index.js` switches which configuration we're using based on the environment we're in. Add a field in there called `logging` or `winston` or something, and put this config in there. You'll need to modify `development.js`, `production.js`, and `test.js`.

At the top of this file, add `const config = require('../../config')`, then you can pass the configuration you defined into into these settings here using `config.logging` (or whatever you named it). You could (and should) go one step further and have `logging.info` and `logging.debug` settings.